package.json

```diff
@@ -57,7 +57,7 @@
   },
   "scripts": {
     "clean": "rimraf dist",
-    "build": "yarn clean && yarn ts:check && tsup",
+    "build": "yarn clean && yarn ts:check && NODE_OPTIONS='--max-old-space-size=8192' tsup",
```

Comment:
This project is not that big. I noticed that the build is very fast; however, the DST generation is very slow. Could you look in the TSUP documentation to see if there is a way to speed it up?